app/src/main/java/uk/gov/onelogin/appcheck/AppIntegrityResult.kt

```diff
@@ -1,7 +1,7 @@
 package uk.gov.onelogin.appcheck
 
 sealed class AppIntegrityResult {
-    data class Success(val firebaseJwt: String) : AppIntegrityResult()
+    data class Success(val proofOfPossession: String) : AppIntegrityResult()
```

Comment:
I'm not sure this is the proof of possession?

Reply:
I've changed this as we talked earlier